assignments/arrays.js

```diff
@@ -63,32 +63,73 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`This is a ${inventory[32].car_model} made by ${inventory[32].car_make} in the year ${inventory[32].car_year} ` );
+
+`or`
+
+console.log(inventory[32])
+
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
+
+  for (let i = 0; i < inventory.length; i++){
+      if (inventory[i] === inventory[inventory.length - 1]){
+        console.log(inventory[i]);
+      }
+  }
+  
 console.log();
 
+
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for(i = 0; i < inventory.length; i++){
+  let modelList = inventory[i].car_model
+  carModels.push(modelList)
+  carModels.sort()
```

Comment:
In terms of efficiency, I recommend to use the sort method after the loop has run through all iterations and the carModels array is finalized instead of sorting every iteration